Gemfile

```diff
@@ -28,6 +28,8 @@ gem 'ancestry', '~> 3.0'
 gem 'rails_admin', '~> 1.4'
 gem 'rails_admin_nestable', '~> 0.3.2'
```

Comment:
Should this PR remove `rails_admin` as well?

Reply:
I'm keeping that for reference (at the small price of mounting the engine in the routes, I can still look at the existing one for reference).
I was thinking to remove this and all the related files in a separate PR once we've replicated everything it's in now, so:
* PART 3 will be replicating the file upload as it is now, and 
* PART 4 removing this and everything else related to RailsAdmin

Would that be acceptable for you?